tests/storage/export.go

```diff
@@ -635,12 +635,12 @@ var _ = SIGDescribe("Export", func() {
 	}
 
 	matchesCNOrAlt := func(cert *x509.Certificate, hostName string) bool {
-		fmt.Fprintf(GinkgoWriter, "CN: %s, hostname: %s\n", cert.Subject.CommonName, hostName)
+		logToGinkgoWritter("CN: %s, hostname: %s\n", cert.Subject.CommonName, hostName)
```

Comment:
maybe there is something special I am not aware of why this writes directly to the ginkgo writer (like maybe due to the package nesting), but it is in general possible to use our default logger. It redirects to the ginkgo writer in tests. Since it does not return an error, it may make this helper function obsolete though.

Reply:
I also can't reason why this was originally logging to the ginkgo writer.

Reply:
I don't think it matters at the moment. The important part is that it does not just log to stdout.